src/utils/DockerMachineUtil.js

```diff
@@ -157,6 +157,14 @@ var DockerMachine = {
         util.exec([path.join(process.env.RESOURCES_PATH, 'terminal'), `DOCKER_HOST=${machineUrl} DOCKER_CERT_PATH=${path.join(util.home(), '.docker/machine/machines/' + machineName)} DOCKER_TLS_VERIFY=1 ${cmd}`]).then(() => {});
       });
     }
+  },
+  virtualBoxLogs: function (machineName = this.name()) {
+    let logsPath = path.join(util.home(), '.docker', 'machine', 'machines', machineName, machineName, 'Logs', 'VBox.log');
+    if (fs.existsSync(logsPath)) {
```

Comment:
I think npm recommends doing a try/catch on readFileSync instead of doing an exist check.
```
let logData = null;
try {
  logData = fs.readFileSync(logsPath, 'utf8');
} catch (e) {
  console.error(e);
}
return logData;
```